frontend/src/app/_services/util.service.ts

```diff
@@ -42,15 +41,13 @@ export class UtilService {
     return id;
   }
 
-  handleError(error: Response | any) {
-    let body;
+  handleError(error: HttpErrorResponse | any) {
     let errors: any = [];
-    if (error instanceof Response) {
+    if (error instanceof HttpErrorResponse) {
       if (error.status) {
         switch (error.status) {
           case 400:
-            body = error.json() || '';
-            errors = body.errors;
+            errors = error.error.errors || '';
```

Comment:
is this how angular 5's `HttpErrorResponse` handles responses? That deeply nested?